src/peergos/shared/io/ipfs/api/Peer.java

```diff
@@ -0,0 +1,33 @@
+package peergos.shared.io.ipfs.api;
+
+import peergos.shared.io.ipfs.cid.*;
+import peergos.shared.io.ipfs.multiaddr.*;
+import peergos.shared.io.ipfs.multihash.*;
+
+import java.util.*;
+import java.util.function.*;
+
+public class Peer {
+    public final MultiAddress address;
+    public final Multihash id;
+    public final long latency;
+    public final String muxer;
+    public final Object streams;
+
+    public Peer(MultiAddress address, Multihash id, long latency, String muxer, Object streams) {
```

Comment:
Perhaps add a docstring explaining what eacch thing is used for? For example  it  would  help to  know that ``muxer``  is  in fact a label to something, rather than a multiplexer itself.